doc/explanations/exchange-scenarios.rst

```diff
@@ -0,0 +1,244 @@
+=========================================
+Exchange multi-asset management scenarios
```

Comment:
This looks like some really excellent documentation, but it is uses a lot of components downstream of the ledger:
* cardano-graphql
* cardano-rosetta
* cardano-db-sync
* cardano-wallet

The folks working in this repository (cardano-ledger) will not know when things fall out of date, and this documentation will very likely quickly become invalid if it lives here.

Reply:
Noted, thanks! I'll delete this explainer from the ledger repo and see where we can host it.